packages/server/test/node.spec.ts

```diff
@@ -386,6 +392,136 @@ describe('Node Specific Cases', () => {
           expect(response.status).toBe(200);
           await expect(response.text()).resolves.toBe('Hello world!');
         });
+
+        it('responds with the correct status code', async () => {
+          const expected: { status: number; statusText: string }[] = [];
+          const received: { status: number; statusText: string; resText?: string }[] = [];
+          for (const statusCodeStr in STATUS_CODES) {
```

Comment:
should these cases be shared only on Node or also on other environments?

Reply:
In other environments(non Node), they use `Request/Response`(Fetch API) already, and we use their native Fetch API implementation.